integrationtests/pom.xml

```diff
@@ -26,49 +26,51 @@
     </properties>
 
     <dependencies>
+
+        <!-- project dependencies - test -->
         <dependency>
             <groupId>io.kroxylicious</groupId>
             <artifactId>kroxylicious</artifactId>
-            <version>0.3.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>io.kroxylicious</groupId>
-            <artifactId>kroxylicious-test-tools</artifactId>
-            <version>0.3.0-SNAPSHOT</version>
+            <artifactId>kroxylicious-multitenant</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>io.kroxylicious</groupId>
-            <artifactId>kroxylicious-multitenant</artifactId>
-            <version>0.3.0-SNAPSHOT</version>
+            <artifactId>kroxylicious-test-tools</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>io.kroxylicious</groupId>
             <artifactId>kroxylicious-schema-validation</artifactId>
-            <version>0.3.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>io.kroxylicious.testing</groupId>
-            <artifactId>testing-junit5-extension</artifactId>
+            <artifactId>testing-api</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>io.kroxylicious.testing</groupId>
-            <artifactId>testing-api</artifactId>
+            <artifactId>testing-junit5-extension</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!-- project dependencies - test -->
```

Comment:
duplicated `project dependencies - test`, is this meant to be `third party dependencies - runtime and compile`?